Sources/Networking/Core/DownloadAPIManager.swift

```diff
@@ -34,7 +34,13 @@ open class DownloadAPIManager: NSObject, Retryable {
         errorProcessors: [ErrorProcessing] = []
     ) {
         /// generate session id in readable format
-        sessionId = Date().ISO8601Format()
+        if #unavailable(iOS 15) {
```

Comment:
Why is the if sometimes `#available` and sometimes `#unavailable` if they both have `else`?

Reply:
Just for feeling of readability. Available I used to hide feature unavailable to see immediately there is fall back to old versions